src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int defaultCircleRadius = 10;
+    private final String defaultCircleColor = "white";
+    private final int figureCount = 5;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int num = random.nextInt(figureCount);
+
+        switch (num) {
+            case 0:
+                return new Square(random.nextInt(100));
+            case 1:
+                return new Circle(random.nextInt(100));
+            case 2:
+                return new Rectangle(random.nextInt(100),
+                        random.nextInt(100));
+            case 3:
+                return new RightTriangle(random.nextInt(100),
+                        random.nextInt(100));
+            case 4:
+                return new IsoscelesTrapezoid(random.nextInt(100),
+                        random.nextInt(100),
+                        random.nextInt(100));
```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)